src/components/SignIn/SignInForm.tsx

```diff
@@ -109,7 +106,7 @@ const SignInForm = (): React.JSX.Element => {
 			</Button>
 
 			<div>
-				<a href={`${apiUrl}forgot-passsword`} className='do-not-remember-password'>I don't remember my password</a>
+				<a href='pages/forgot-passsword' className='do-not-remember-password'>I don't remember my password</a>
```

Comment:
```suggestion
				<a href='/pages/forgot-passsword' className='do-not-remember-password'>I don't remember my password</a>
```

It is missing the leading slash, without it the url will be relative from the current url and not absolute.